trilitech/tezedge
# Remove non-required crates

Only `crypto`, `tezos_encoding`, & `tezos_encoding_derive` crates will be maintained for use in the **Smart Rollup Kernel SDK**

# Manually testing

Clone & switch to the `emturner@remove-unused` branch.  rust-1.66 should be installed, and then run the following:

```
cargo check 
cargo test
```
